timescaledb/getting-started/query-data.md

```diff
@@ -1,153 +1,135 @@
-# 5. Query your data
+# Query your data
 
 With TimescaleDB, there's no need to learn a custom query language. **TimescaleDB
-supports full SQL**. This means you can put your SQL knowledge to good use and
-use the rich ecosystem of PostgreSQL tools you know and love.
-
-For example, here's how to find the average temperature for each city in the past 2 years:
-
-```sql
---------------------------------
--- Average temperature per city
--- in past 2 years
---------------------------------
-SELECT city_name, avg(temp_c)
-FROM weather_metrics
-WHERE time > now() - INTERVAL '2 years'
-GROUP BY city_name;
-```
-
-And here's how to find the total snowfall for each city in the past 5 years:
-
-```sql
---------------------------------
--- Total snowfall per city
--- in past 5 years
---------------------------------
-SELECT city_name, sum(snow_1h_mm)
-FROM weather_metrics
-WHERE time > now() - INTERVAL '5 years'
-GROUP BY city_name;
-```
-
-
-<highlight type="tip">
-Fun fact: TimescaleDB adds important enhancements to the PostgreSQL query planner
-that improve query reusability for INTERVAL predicates, something PostgreSQL does
-not have.
-</highlight>
-
-## Advanced SQL functions for time-series data
+supports full SQL**. You can use your SQL knowledge along with the rich 
+ecosystem of PostgreSQL tools, and add the extra features and performance of
+TimescaleDB.
+
+
+Here are some query examples so you can get familiar with using SQL alongside
+popular TimescaleDB functions.
+
+## Basic SQL queries
+### Select all stock data from the last day.
+
+   This query uses the [`WHERE`][clause-expressions] clause to only show results for
+   timestamps within the last day. 
+
+   ```sql
+   SELECT * FROM stocks_real_time srt
+   WHERE time > now() - INTERVAL '1 day';
+   ```
+
+### Select the top 10 stock values by price.
+
+   This query uses the [`ORDER BY`][order-by] keyword to order the results by descending price
+   and limit the number of results shown to 10. 
+
+   ```sql
+   SELECT * FROM stocks_real_time srt
+   ORDER BY price DESC
+   LIMIT 10;   
+   ```
+
+### Calculate the average trade price for Apple from the last day
+
+   This query uses the [`avg()`][average] function along with [`FILTER`][filter] and `WHERE` 
+   to include only Apple trades made within the last day. Instead of using Apple's symbol, 
+   the [`JOIN`][join] operator allows you to use the company name.  
+
+   ```sql
+   SELECT
+   avg(price) FILTER (WHERE c.name = 'Apple')
+   FROM stocks_real_time srt
+   JOIN company c ON c.symbol = srt.symbol
+   WHERE time > now() - INTERVAL '2 day';
```

Comment:
```suggestion
 SELECT 
  avg(price) avg_price 
FROM 
  stocks_real_time srt 
  JOIN company c ON c.symbol = srt.symbol 
WHERE 
  c.name = 'Apple' 
  AND time > now() - INTERVAL '2 day';
```
Filter isn't a common way to query this and it doesn't add specific value here. I'd keep it simple and build.